content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -276,34 +274,37 @@ once bound to a node, a Pod will never be rebound to another node.
 
 ## Pod lifetime
 
-In general, Pods remain until a human or controller process explicitly removes them.
+In general, Pods remain until a human or
+{{< glossary_tooltip term_id="controller" text="controller" >}} process
+explicitly removes them.
 The control plane cleans up terminated Pods (with a phase of `Succeeded` or 
-`Failed`), when the number of Pods exceeds the configured threshold 
+`Failed`), when the number of Pods exceeds the configured threshold
 (determined by `terminated-pod-gc-threshold` in the kube-controller-manager).
 This avoids a resource leak as Pods are created and terminated over time.
 
-Three types of controllers are available:
+There are different kinds of resources for creating Pods:
 
-- Use a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) for Pods that are expected to terminate,
+- Use a {{< glossary_tooltip term_id="deployment" >}},
+  {{< glossary_tooltip term_id="replica-set" >}} or {{< glossary_tooltip term_id="statefulset" >}}
+  for Pods that are not expected to terminate, for example, web servers.
+
+- Use a {{< glossary_tooltip term_id="job" >}}
+  for Pods that are expected to terminate once their work is complete;
   for example, batch computations. Jobs are appropriate only for Pods with
   `restartPolicy` equal to OnFailure or Never.
 
-- Use a [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/),
-  [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/), or
-  [Deployment](/docs/concepts/workloads/controllers/deployment/)
-  for Pods that are not expected to terminate, for example, web servers.
-  ReplicationControllers are appropriate only for Pods with a `restartPolicy` of
-  Always.
-
-- Use a [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) for Pods that need to run one per
-  machine, because they provide a machine-specific system service.
+- Use a {{< glossary_tooltip term_id="daemonset" >}}
+  for Pods that need to run one per computer in your cluster, because they
+  provide a machine-specific system service.
 
-All three types of controllers contain a PodTemplate. It
-is recommended to create the appropriate controller and let
-it create Pods, rather than directly create Pods yourself. That is because Pods
-alone are not resilient to machine failures, but controllers are.
+All of those different workload resources contain a PodTemplate. It
+is recommended to create the appropriate resource and let
+it create Pods, rather than directly create Pods yourself. On their own,
+Pods are not resilient to machine failures. Workload resources such
+as ReplicaSet have a control loop that is responsible for recovering from a
+Node- or Pod-level failure.
```

Comment:
suggest removing this paragraph..

Reply:
I'm concerned that removing it means the reader doesn't get an opportunity to learn that Pods aren't resilient to machine failures. That seems like an important point.
@mikebrow do you see what I mean?

Reply:
Yes, but it introduces an entirely new very large topic, machine failures, without explaining the types of machine failures or what is meant by machine failures. A machine failure does not change the pod's lifecycle but it may trigger its abrupt end. Pods will be resilient to some/many types of machine failures. Does not touch on container failures or pod specific resources.

I know what you mean here but it introduces more questions than it addresses. Is there a place in the docs where node/pod/container failover is discussed talking about pod resiliency? If so could have a Note: comment that links to it?